test/fetchival.js

```diff
@@ -0,0 +1,45 @@
+const test = require('tape')
+const { fetchival } = require('../index')
+
+function Captor() {
+  const captor = {
+    calls: [],
+    async capture(...args) {
+      this.calls.push(args);
+      return { status: 204 }
+    }
+  }
+  const capture = captor.capture.bind(captor)
+  capture.calls = captor.calls
+
+  return capture;
+}
```

Comment:
it's sort of a custom `jest.fn()` implementation, so `const captor = Captor()` can be used the same as `const mock = jest.fn()` and passed instead of the real thing. And then you assert against `captor.calls`